Makefile

```diff
@@ -310,7 +311,8 @@ ifneq ($(RESTY_IMAGE_BASE),src)
 	KONG_ADMIN_URI="http://$(TEST_HOST):$(TEST_ADMIN_PORT)" \
 	KONG_PROXY_URI="http://$(TEST_HOST):$(TEST_PROXY_PORT)" \
 	TEST_COMPOSE_PATH=$(TEST_COMPOSE_PATH) \
-	KONG_BASE_IMAGE_NAME=$(DOCKER_REPOSITORY):openresty-$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_OPENRESTY_SUFFIX) \
+	DOCKER_GO_BUILDER=$(DOCKER_REPOSITORY):openresty-$(RESTY_IMAGE_BASE)-$(RESTY_IMAGE_TAG)-$(DOCKER_OPENRESTY_SUFFIX) \
+	KONG_BASE_IMAGE_NAME=$(KONG_TEST_IMAGE_NAME) \
```

Comment:
I believe this is no longer used.

Reply:
`DOCKER_GO_BUILDER` is the openresty image we build go in
`KONG_BASE_IMAGE_NAME` is the Kong docker image built from our docker-kong Dockerfile

the `03-go-plugins/up.sh` builds the go plugin using one then uses the other to cp the go plugin into and test it

Reply:
I might be missing something, but `KONG_BASE_IMAGE_NAME` isn't actually used anywhere, is it? I can only see one reference to it, and it is an `echo` that could be removed.

Reply:
yup I see what happened I thought `KONG_BASE_IMAGE_NAME==KONG_TEST_IMAGE_NAME`